modules/footer/Footer.tsx

```diff
@@ -1,16 +1,18 @@
 import Image from 'next/image';
+import styles from './Footer.module.css';
+
 export const Footer = () => {
   return (
-    <footer className="flex justify-between items-center my-8">
-      <a className="block" href="https://vercel.com?utm_source=lyonjs&utm_campaign=oss">
+    <footer className={styles.footer}>
+      <span>@LyonJS</span>
```

Comment:
Do you want to add a link here ? Like Twitter for instance ? If not, maybe without the `@` it would look less like a link